rfc8446.xml

```diff
@@ -4372,11 +4375,13 @@ streamed to the server over a longer time period.</t>
 
 <section anchor="mandatory-to-implement-cipher-suites" title="Mandatory-to-Implement Cipher Suites">
 
-<t>In the absence of an application profile standard specifying otherwise, a
+<t>In the absence of an application profile standard specifying otherwise:</t>
+
+<t>A
```

Comment:
The second paragraph (now third) of this section presumably also only applies in the absence of an application profile.